tests/catalyst/tests/test_finetune2.py

```diff
@@ -115,6 +129,16 @@ def get_callbacks(self, stage: str):
             callbacks["confusion_matrix"] = dl.ConfusionMatrixCallback(
                 input_key="logits", target_key="targets", num_classes=10
             )
+            callbacks["f1_score"] = dl.SklearnCallback(
+                keys={
+                    "y_pred": "labels",
+                    "y_true": "targets",
+                    "average": "micro",
+                    "zero_division": 1,
+                },
+                metric_fn="f1_score",
+                metric_key="f1_score",
+            )
```

Comment:
could you please add this example to the callbacks docs? it's very interesting, that you could also pass metrics **kwargs through the keys